helm-chart/images/image-cleaner/image-cleaner.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+import os
+import sys
+import docker
+import time
+
+def get_inodes_available_fraction(path):
+    """
+    Return available_inodes / total_inodes in device containing path
+    """
+    stat = os.statvfs(path)
+    return float(stat.f_favail) / float(stat.f_files)
+
+def get_docker_images(client):
```

Comment:
It seems like (in a future PR) we could run a query to Prometheus to sort images by popularity as well. Not saying we should do this on the current PR though